traffic_ops/testing/api/v4/logs_test.go

```diff
@@ -46,3 +47,20 @@ func GetTestLogsByLimit(t *testing.T) {
 		t.Fatalf("Get logs by limit: incorrect number of logs returned (%d)", len(toLogs.Response))
 	}
 }
+
+func GetTestLogsByUsername(t *testing.T) {
+	opts := client.NewRequestOptions()
+	opts.QueryParameters.Set("username", "admin")
+	toLogs, _, err := TOSession.GetLogs(opts)
+	if err != nil {
+		t.Errorf("error getting logs: %v - alerts: %+v", err, toLogs.Alerts)
+	}
+	for _, user := range toLogs.Response {
+		if *user.User != "admin" {
+			t.Fatalf("incorrect username seen in logs, expected: `admin`, got: %v", *user.User)
+		}
+	}
+	if len(toLogs.Response) <= 0 {
+		t.Fatalf("Get logs by username: incorrect number of logs returned (%d)", len(toLogs.Response))
```

Comment:
So, I am running the `go test` for logs only and without doing anything additional, I am able to see 10 logs for the user since the first statement of the TestLogs() is:
```
WithObjs(t, []TCObj{Roles, Tenants, Users}, func() { // Objs added to create logs when this test is run alone
```

So, the probability that no logs will be present is 0.

Reply:
:+1: